go.mod

```diff
@@ -76,7 +80,22 @@ require (
 	google.golang.org/grpc v1.45.0
 )
 
+require github.com/aws/aws-sdk-go-v2/config v1.15.5
```

Comment:
should be moved in the previous require block

Reply:
Fixed. I should do something wrong when I update mod / vendor :(